napari/layers/utils/layer_utils.py

```diff
@@ -828,11 +830,6 @@ def remove(self, indices: Any) -> None:
         indices : Any
             The indices of the rows to remove. Must be usable as the labels parameter
             to pandas.DataFrame.drop.
-
-        Returns
```

Comment:
Unrelated to this PR, but should be removed as this method does not return anything.